kickoff/jsonexport.py

```diff
@@ -228,6 +228,13 @@ def updateLocaleWithVersionsTable(product):
     buildsVersionLocales = generateLocalizedBuilds(buildsVersionLocales,
                                                    esr_releases[0][2],
                                                    esr_releases[0][0] + "esr")
+
+    esr_next = getFilteredReleases(product, ["esr"], lastRelease=True, withL10N=True, esrNext=True)
+    if esr_next:
+        buildsVersionLocales = generateLocalizedBuilds(buildsVersionLocales,
+                                                       esr_next[0][2],
+                                                       esr_next[0][0] + "esr")
```

Comment:
if config.ESR_NEXT == '', we won't pass `if esr_next`. I know this looks like a spaghetti plate. I'll try to clean that up in a follow up